app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/CreateSuggestion.kt

```diff
@@ -335,14 +335,27 @@ fun CreateSuggestion(
                                   website = _website,
                                   imageUrl = ""))
                   // Pass the created suggestion to the callback function
-                  if (viewModel.addSuggestion(tripId, suggestion)) {
-                    onSuccess()
+                  if (suggestion.suggestionId != "") {

```

Comment:
Note : you can use `is(Not)Empty` method to check if a string is empty/not empty.